libs/src/services/creatorNode/index.js

```diff
@@ -209,22 +204,17 @@ class CreatorNode {
   async uploadTrackMetadata (metadata, sourceFile) {
     // this does the actual validation before sending to the creator node
     // if validation fails, validate() will throw an error
-    try {
-      this.schemas[SchemaValidator.trackSchemaType].validate(metadata)
-      console.log('track metadata validation passed', metadata)
+    this.schemas[SchemaValidator.trackSchemaType].validate(metadata)
+    console.log('track metadata validation passed', metadata)
```

Comment:
if `validate()` fails, there will be a `console.error(..)` message. then, when that finishes, this path will log `console.log('...passed')`. i think we need to add some logic here to cover if validate fails